src/SFA.DAS.RoATPService.Data/OrganisationRepository.cs

```diff
@@ -0,0 +1,136 @@
+﻿namespace SFA.DAS.RoATPService.Data
+{
+    using System;
+    using System.Data;
+    using System.Data.SqlClient;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Application.Interfaces;
+    using AssessorService.Data.DapperTypeHandlers;
+    using Dapper;
+    using Domain;
+    using Settings;
+
+    public class OrganisationRepository : IOrganisationRepository
+    {
+        private IWebConfiguration _configuration;
+
+        public OrganisationRepository(IWebConfiguration configuration)
+        {
+            _configuration = configuration;
+            SqlMapper.AddTypeHandler(typeof(OrganisationData), new OrganisationDataHandler());
```

Comment:
Another FYI.  There is a new field going into the 'OrganisationData' in assessor-service, string TradingName.  The story with that is about to go into testing (ON-1469). We're going to have to keep an eye on all these common objects.  Have a Type library at some point, I guess.